scripts/test/Muon/elemental_analysis/periodic_combo_test.py

```diff
@@ -0,0 +1,77 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+from __future__ import print_function, absolute_import
+
+import unittest
+
+from mantid.py3compat import mock
+from mantidqt.utils.qt.testing import start_qapplication
+
+import Muon.GUI.ElementalAnalysis.PeriodicTable.periodic_table as periodic_table
+from Muon.GUI.ElementalAnalysis.PeriodicTable.periodic_table import PeriodicTableItem, PeriodicCombo
+
+
+@start_qapplication
+class PeriodicComboTest(unittest.TestCase):
```

Comment:
what do we mean by periodic combo?

Reply:
It's a `QComboBox` that lists all elements of the periodic table:
https://github.com/mantidproject/mantid/blob/e383f53ca60873572815ef90ea75d596fc71b153/scripts/Muon/GUI/ElementalAnalysis/PeriodicTable/periodic_table.py#L625